README.md

```diff
@@ -78,37 +78,47 @@ You might not have to sort the output, if you run through reads
 
 Implement the two algorithms in two tools, `naive` and `lin`, that must be present at the root of the repository once they are built. The test setup checks that they give the correct output on selected data, but you should still carefully test them.
 
-Once you have implemented the tools, fill out the report below. 
+Once you have implemented the tools, fill out the report below.
 
 ## Report
 
-### Insights you may have had while implementing and comparing the algorithms. 
+### Insights you may have had while implementing and comparing the algorithms.
 
-*Describe this here.*
+There are many ways to archive the same thing but not every method is equally elegant.
 
-### Problems encountered if any. 
+### Problems encountered if any.
 
-*Describe this here.*
+* Plotting resonable data can be very hard. Mainly making the plots readable so a conclusion can be made from them is difficult.
+* Indices can also be difficult.
 
 ### Experiments that verifies the correctness of your implementations.
 
-*Describe this here.*
+We have made testing with some examples were we knew the correct answer and checked that the algorithm gave the same answer as we expected.
+
+After being confident in that the naive algorithm as correct we also ran alot of examples with randomly generated data and compared the output given by different algorithms to make sure they all gave the same answer to the same input.
```

Comment:
Random strings have peculiar properties that might not be shared with other kinds of strings. It is risky to rely too much on one class of strings...